### PR TITLE
feat: new component Stack

### PR DESCRIPTION
**Pull request by Exayy**
## Type

- Bug
- Feature ✅ 
- Enhancement
- Documentation
- Migration
- Refactor

#### What is expected?

Add a new Stack component

### CODE REVIEW

**Review by baptisteArno — APPROVED** [review]
❤️

File: src/components/Stack/index.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  shouldForwardProp: prop =>
+    !['gap', 'direction', 'alignItems', 'justifyContent'].includes(prop),
+})<StackCSSProperties>`
+  display: flex;
+
+  ${({ theme, gap, direction, alignItems, justifyContent }) => `
+    gap: ${gap * theme.space.unit}px;
+    flex-direction: ${direction};
+    align-items: ${alignItems || 'normal'};
+    justify-content: ${justifyContent || 'normal'};
```
Comment (baptisteArno) [inline src/components/Stack/index.tsx:26]: 'normal' doesn't seem to be a valid property?

File: src/components/Stack/index.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  `}
+`
+
+const Stack = ({
```
Comment (chambo-e) [inline src/components/Stack/index.tsx:30]: useless component no ?

Comment (Exayy) [reply src/components/Stack/index.tsx]: Indeed  👌

File: src/components/Stack/index.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+const Stack = styled('div', {
+  shouldForwardProp: prop =>
+    !['gap', 'direction', 'alignItems', 'justifyContent'].includes(prop),
+})<StackProps>`
+  display: flex;
+
+  ${({ theme, gap = 0, direction = 'column', alignItems, justifyContent }) => `
+    gap: ${theme.space[gap]};
+    flex-direction: ${direction};
+    align-items: ${alignItems || 'normal'};
```
[the remark this answers is not in the document]
Comment (chambo-e) [reply src/components/Stack/index.tsx:23]: Alright, nvm, seems that default is normal :)

File: src/components/Stack/index.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  children: ReactNode
+}
+
+const Stack = styled('div', {
+  shouldForwardProp: prop =>
+    !['gap', 'direction', 'alignItems', 'justifyContent'].includes(prop),
+})<StackProps>`
+  display: flex;
+
+  ${({ theme, gap = 0, direction = 'column', alignItems, justifyContent }) => `
```
Comment (chambo-e) [inline src/components/Stack/index.tsx:20]: gap should be a string no ?

Comment (chambo-e) [inline src/components/Stack/index.tsx:20]:
```suggestion
  ${({ theme, gap = 0, direction = 'column', alignItems = 'normal', justifyContent = 'normal' }) => `
```

Comment (chambo-e) [reply src/components/Stack/index.tsx:20]: Apparently no 🤔

**Review by chambo-e — APPROVED** [review]

**Pull request merged**